### PR TITLE
Completed Javascript Loops - Amelia Talkington

### PR DESCRIPTION
**Pull request by ameliatalkington**
Screen Shot 2021-01-28 at 10 14 55 AM

### CODE REVIEW

File: javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
+  for (currentNumber = 1; currentNumber <= 10; currentNumber++) {
+    numbers.push(currentNumber);
+  }
```
Comment (codingcodymiller) [inline javascript-loops/main.js:14]: This should be a `while` loop

File: javascript-loops/main.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  return numbers;
+}
+
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  var currentNumber = 2;
+  for (currentNumber = 2; currentNumber <= 20; currentNumber += 2) {
+    evenNumbers.push(currentNumber);
+  }
```
Comment (codingcodymiller) [inline javascript-loops/main.js:23]: This should be a `while` loop

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far!! A couple minor notes:

**Comment by ameliatalkington** [comment]
Screen Shot 2021-01-28 at 1 03 29 PM
Changed to while loops

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**